src/main.rs

```diff
@@ -64,6 +64,12 @@ struct Args {
 
     #[command(subcommand)]
     command: Commands,
+
+    #[arg(short, long)]
+    features: Vec<String>,
+
+    #[arg(long)]
+    all_features: bool,
```

Comment:
Would there be value in also having a `--no-default-features` the same as other `cargo` commands with features?

Should these should go before `config` above, or does that produce an unnatural ordering in the command-line help? They could use a short doc comment for that in any case.